Sources/SPMBuildCore/BinaryTarget+Extensions.swift

```diff
@@ -89,14 +89,41 @@ fileprivate extension Triple {
     }
 }
 
-fileprivate extension Triple.OS {
-    /// Returns a representation of the receiver that can be compared with platform strings declared in an XCFramework.
-    var asXCFrameworkPlatformString: String? {
-        switch self {
-        case .darwin, .linux, .wasi, .windows, .openbsd:
-            return nil // XCFrameworks do not support any of these platforms today.
-        case .macOS:
-            return "macos"
+fileprivate extension Triple {
+    var asXCFrameworkPlatformString: (String, String)? {
```

Comment:
Can you preserve the doc comment that's currently in this diff was removed and update it accordingly to clarify what each value of the tuple means? We may also not want to model this is a tuple, but as a `struct`.